plenum/common/func_utils.py

```diff
@@ -0,0 +1,28 @@
+
+
+def flatten(some_list):
+    """
+    [[a,b],[c,d]] => [a,b,c,d]
+    """
+    for inner_list in some_list:
+        for item in inner_list:
+            yield item
+
+
+def decons(some_list):
+    """
+    [a,b,c,d] => (a, [b,c,d])
+    """
+    if len(some_list) < 2:
+        return None
+    return some_list[0], some_list[1:]
+
+
+def count(some_list):
```

Comment:
I want a more clear function name than "decons" for this. It should be obvious from a function name, what it does. Having a good example comment is also desirable, but does not eliminate the need for a good name.